src/todo.js

```diff
@@ -0,0 +1,48 @@
+const todos = [];
+
+function loadTasks() {
+  const savedTasks = localStorage.getItem('tasks');
+  if (savedTasks) {
+    todos.length = 0;
+    todos.push(...JSON.parse(savedTasks));
+  }
+}
+
+function updateIndexes() {
+  todos.forEach((task, index) => {
+    task.index = index;
+    // console.log(task);
+  });
+}
+
+function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: todos.length,
+  };
+
+  todos.push(task);
+  localStorage.setItem('tasks', JSON.stringify(todos));
+}
+
+function deleteTask(index) {
+  todos.splice(index, 1);
+
+  updateIndexes();
+
+  localStorage.setItem('tasks', JSON.stringify(todos));
+}
```

Comment:
- [ ] Please make sure the expected task is deleted from the list instead of the one at the top


https://github.com/kerlos-bahaa/to-do-list/assets/44798044/0e90d7c6-e745-4e9c-a84c-dfd0eaade8d4